guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -8,8 +8,8 @@ functional_areas:
   - Setup
   - Configuration
 ---
-
-<!-- Assigning liquid variables for placeholder values
+<!--MAGECLOUD
```

Comment:
What is the purpose of this block of liquid tags?

Reply:
The tags and values look like circular references. I don't understand the utility of these. Why not just use plain text? The tags don't appear to offer any convenience. Their presence is confusing.

Reply:
The intent was to show the actual values that cause the deployment error.  Those values are present in the error message. That's why I included them as is.

Reply:
@meker12 
I didn't recommend it, I shown how to use it.